programmers-coding-test/minjae/Level1/핸드폰-번호-가리기.js

```diff
@@ -0,0 +1,8 @@
+function solution(phone_number) {
+    const starNum = phone_number.split('').slice(-4);
+    
+    for (let i = 0; i < phone_number.length - 4; i++) {
```

Comment:
-4는 프로그램 구현에 절대적인 값이므로 상수로 선언해주는 것이 좋습니다.
변수 선언 케이스도 camelCase로 일치시켜주세요 :) 
혹시 snake_case로 선언해준 이유가 별도로 있나요?

```suggestion
count MOBILE_BACK_NUMBER_LENGTH = -4;

function solution(phoneNumber) {
    const starNum = phoneNumber.split('').slice(MOBILE_BACK_NUMBER_LENGTH);
    const phoneNumberLength = phoneNumber.length - MOBILE_BACK_NUMBER_LENGTH;
    
    for (let i = 0; i < phoneNumberLength; i++) {
```

Reply:
나는 이거때메 이렇게 했는데 왜 니 맘대로 바꾸냐